.github/actions/file-diff/index.js

```diff
@@ -78,25 +82,30 @@ async function run() {
 		const markdown = [];
 		const summary = [
 			"### Summary",
-			`**Total size**: ${bytesToSize(overallDiffSize)}<sup>*</sup>`
+			`**Total size**: ${bytesToSize(overallHeadSize)}<sup>*</sup>`,
 		];
 		const summaryTable = [];
 
 		if (sections.length === 0) {
 			summary.push(...["", " 🎉 No changes detected in any packages"]);
 		} else {
-			if (diffOutput.size > 0 && hasDiff) {
-				let changeSummary = `**Total change (Δ)**: ${printChange(
-					overallDiffSize - overallSize
-				)}`;
-
-				if (overallSize === overallDiffSize) changeSummary += " 🎉";
-				else
-					changeSummary += ` (${printPercentChange(
-						(overallDiffSize - overallSize) / overallSize
-					)})`;
-
-				summary.push(...[changeSummary, ""]);
+			/**
+			 * Calculate the change in size
+			 * PR - base / base = change
+			 */
+			let changeSummary = "";
+			if (baseOutput.size > 0 && hasDiff && hasChange) {
+				changeSummary = `**Total change (Δ)**: ${printChange(overallHeadSize, overallBaseSize)} (${printPercentChange(overallHeadSize, overallBaseSize)})`;
+			} else if (baseOutput.size > 0 && hasDiff && !hasChange) {
+				changeSummary = `No change in file sizes`;
+			}
+
+			if (changeSummary !== "") {
+				summary.push(...[
+					changeSummary,
+					"Table below reports only on changes to a package's main file. Other changes can be found in the collapsed \"Details\" below.",
+					""
+				]);
```

Comment:
I'm not seeing this note in this PR or the [storybook PR](https://github.com/adobe/spectrum-css/pull/2398) this was tested on 🤔 Am I missing something?

Reply:
Ah it looks like I added the note after I had merged & run this on the sub-branch.  I just pulled in the latest so it's running now.

Reply:
Cool! I see it in the Storybook PR now 👍